docs/api/ChampionMasteryManager.md

```diff
@@ -141,8 +158,6 @@ highest(n?: number): Promise<ChampionMastery>;
 
 #### .refreshAll ()
 
-Update the cache with the latest data for all champions' mastery data for this summoner.
-
 
 
```

Comment:
Also it seams like vuepress is not generating docs for deprecated properties

Reply:
The generator is actually manually written but yeah, it is designed to ignore private and deprecated properties when generating docs.

Reply:
privates should be ignored while deprecated  should be marked as such, probably with version when they got deprecated